scripts/deterministic/DeployScroll.s.sol

```diff
@@ -611,6 +640,38 @@ contract DeployScroll is DeterminsticDeployment {
         );
     }
 
+    function deployGasToken() private gasToken(true) {
+        uint8 decimal = 18;
+        string memory key = ".general.EXAMPLE_GAS_TOKEN_DECIMAL";
+        if (vm.keyExistsToml(cfg, key)) {
+            decimal = uint8(cfg.readUint(key));
+        }
+
+        bytes memory args = abi.encode(
+            "ScrollGasToken", // _name
+            "GasToken", // _symbol
+            decimal, // _decimals
+            DEPLOYER_ADDR, // _recipient
+            10**28 // _amount
+        );
+
+        L1_GAS_TOKEN_ADDR = deploy("L1_GAS_TOKEN", type(GasTokenExample).creationCode, args);
```

Comment:
So if the dev does not specify the `L1_GAS_TOKEN` in `config.toml`, this will automatically deploy a test gas token contract, correct?

Reply:
Yes, that's true.